driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/execution/testng/SeleniumRetryAnalyzer.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Testerra
+ *
+ * (C) 2021, Mike Reiche,  T-Systems Multimedia Solutions GmbH, Deutsche Telekom AG
+ *
+ * Deutsche Telekom AG and all other contributors /
+ * copyright owners license this file to you under the Apache
+ * License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package eu.tsystems.mms.tic.testframework.execution.testng;
+
+import java.util.Optional;
+import org.openqa.selenium.json.JsonException;
+import org.openqa.selenium.remote.UnreachableBrowserException;
+
+public class SeleniumRetryAnalyzer implements AdditionalRetryAnalyzer {
```

Comment:
The documentation needs an update about this new default retry analyzer and the WebDriver retry analyzer. The file `https://github.com/telekom/testerra/blob/master/docs/src/docs/execution/execution-retry-analyzer.adoc` in `Specific retries` contains very old stuff from a former framework.

Reply:
Line 52ff still contains some old stuff. Its located in the extract of the `test.properties`

Reply:
Ok, I recognized the `WebDriverRetryAnalyzer` still contains message checks, so there wasn't so much old stuff. Sorry for confusing.